### PR TITLE
links between donations and schools tables

### PR DESCRIPTION
**Pull request by maialeandra**
Created a link on the donations view to the schools view and vice versa. I wonder if there is a more efficient and sustainable way to have the tables linked like having these buttons be revealed if you are not on that page already (similar in logic to having the log out button only appear under the condition of current_admin).

### CODE REVIEW

**Comment by ssalzberg** [comment]
No need to commit the DS_Store files. Those are something local to your machine that don't need to be in the repo. See if you can figure out how to use git to remove those files from version control and push that change to this branch.

https://git-scm.com/docs/git-rm
http://stackoverflow.com/questions/7434449/why-use-git-rm-to-remove-a-file-instead-of-rm

**Comment by ssalzberg** [comment]
Links look good! If you want, you can merge this branch in after deleting the DS_Store files, or you can try make these links more robust by putting them in a "partial" file (http://guides.rubyonrails.org/layouts_and_rendering.html) and rendering them at the top of each page. For example, you could create a file at app/views/shared/_tabs.html.erb and put the donations link and the schools link in it. Then, wherever you want to render those links, you could do:

```
render "shared/tabs"
```

Later you can modify the code in that file to be a bit more tailored, for example by disabling the link for the page that the user is currently on.

**Comment by maialeandra** [comment]
Did I remove the .DS files properly? I also got the render to work, yay! Is this branch ready to be merged with the master branch? I merged the donations-date-column with master so is that going to present a conflict with merging this linked-tables branch now that linked-tables is outdated as compared to master?

File: .gitignore [hunk 1]
```diff
@@ -19,3 +19,8 @@
 
 # Ignore Byebug command history file.
 .byebug_history
+
+# Ignore .DS files
+.DS_Store
+app/assets/.DS_Store
```
Comment (ssalzberg) [inline .gitignore]: You can actually just write:

```
.DS_Store
```

and that will ignore all DS_Store files in all subdirectories.

http://gitready.com/beginner/2009/01/19/ignoring-files.html

**Comment by ssalzberg** [comment]
looks like you removed the ds store files correctly, good job!

after that chagne to the .gitignore, this branch is ready to merge. from the looks of the big green merge button above, looks like there are no conflicts with master.

**Comment by maialeandra** [comment]
Great, thanks! Also, can we delete some of the old branches? What is the standard protocol? My list is just getting a bit cluttered now when I type git branch so if It's common convention to delete branches when we're done, should we do that?

**Comment by ssalzberg** [comment]
yup you can delete branches when everything is merged in. you can do that like this:

```
git branch -D my-branch-name
```

**Pull request merged**